modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1743,6 +1765,104 @@ public void testGetValuesList() throws Exception {
 		Assert.assertEquals(valuesList.toString(), 0, valuesList.size());
 	}
 
+	@Test
+	public void testSaveObjectEntryAsDraft() throws Exception {
+		_objectDefinition.setEnableObjectEntryDraft(true);
+
+		_objectDefinition =
+			_objectDefinitionLocalService.updateObjectDefinition(
+				_objectDefinition);
+
+		Map<String, Serializable> valuesMissingRequiredField =
```

Comment:
This variable is also being used in a lambda expression (So it's value can't be changed). I declared it because I'm using it in two method calls. Thea alternative would be to pass the values (Hash Map) directly.

Reply:
Ok, in this case, I believe we can use `values` and `values1`. What do you thing?

Reply:
Done! I just used `values1` and `values2` as I believe that is the convention